bucket/atom-beta.json

```
@@ -31,14 +31,14 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/atom/atom/releases/download/v$version/atom-x64-$version-full.nupkg",
+                "url": "https://github.com/atom/atom/releases/download/v$version/atom-beta-x64-$version-full.nupkg",
                 "hash": {
                     "url": "$baseurl/RELEASES-x64",
-                    "regex": "([a-fA-F0-9]+)\\s+?(?:atom-$version-full.nupkg)"
+                    "regex": "([a-fA-F0-9]+)\\s+?(?:atom-beta-$version-full.nupkg)"
```

Nonsense unsaved capture group

```suggestion
                    "regex": "$sha1\\s*atom-beta-$version-full.nupkg"
```

---

Yes, totally nonsense...